### ISSUE TITLE
Regular dev call, twice monthly

### ISSUE THREAD

**Issue by max-sixty** [PRQL/prql#1083]
We have a call twice a month for developers; we use this to sync on current discussion issues and, if we have time, broader discussions about the language.

It's at 10.30AM PT on the 1st & 3rd Sunday of every month. It's scheduled for 30-45 mins, and we have a hard-stop at 60 mins. There's an [ics file](https://prql-lang.org/book/project/contributing/fortnightly-dev-call.ics) linked from the [Contribution docs](https://prql-lang.org/book/project/contributing/index.html).

The call is hosted on the [Discord #dev-voice-channel](https://discord.gg/2RUBzz3R3J). All are welcome, including those who haven't yet contributed.

Agenda:

- 60 second updates
- Go through "Priority" items: https://github.com/PRQL/prql/labels/priority
- Go through "Needs Discussion" items: https://github.com/PRQL/prql/labels/needs-discussion
- If we have time, go through "Bug" items: https://github.com/PRQL/prql/labels/bug

We occasionally take some notes at https://hackmd.io/eHmlkHmnTNKbpvyok_iixQ

---

(FYI there are some deleted comments below, that's because @richb-hanover was helpful in making the ics file, and then we wanted to keep this issue clean so removed them)

**Comment by max-sixty** [comment]
To refresh this — we're still on the same schedule, so the next call is on Saturday Jan 8 at 9.30am PT.

As before, the call hosted on the [Discord #dev-voice-channel](https://discord.gg/2RUBzz3R3J). I can send a calendar invite if you message me your email address. All are welcome, including those who haven't yet contributed.

We have been running over the 30 minute slot, but a) please feel completely free to drop off after 30 min, b) I'm really trying to cap at 45 minutes so we keep the conversation focused...

**Comment by richb-hanover** [comment]
Do you mean **SUNDAY** Jan 8 at 9.30am PT?

Or Saturday *Jan 7* at 9.30am PT? Thanks

**Comment by max-sixty** [comment]
Sorry, I had one job, now fixed.

(I'll delete both our messages at some point to keep this issue clean)

**Comment by richb-hanover** [comment]
It would be good to revise this issue's title to reflect the current/upcoming date. (Currently the only info shown in the main Issues page is that it was opened on 25 Oct 2022, so it makes it seem as if it has passed.) Thanks.

**Comment by max-sixty** [comment]
> We meet on Sundays at 9.30AM PT, scheduled for 30 minutes.

Yes but it's every two weeks! I guess we could have "1st and 3rd Sundays of every month"?

**Comment by richb-hanover** [comment]
> 1st and 3rd Sundays of every month

LGTM

And would you put the (upcoming) date in the subject? Thanks.

**Comment by max-sixty** [comment]
> > 1st and 3rd Sundays of every month

Actually it doesn't seem like this is possible in GCal unfortunately

> And would you put the (upcoming) date in the subject?

I'll try and make it more obvious, but I'm not going to remember to change it every time, so I'm hesitant to put the date in

**Comment by richb-hanover** [comment]
I think I've got it... Save the following as "PRQL Dev Call.ics" file. This file creates the repeating event out through the end of March 2023. (See the "UNTIL=...")

NB: If we're to start on 8 Jan 2023, it's the second and fourth Sunday at 9:30am
```
BEGIN:VCALENDAR
PRODID:-//Apple Inc.//macOS 12.6.1//EN
VERSION:2.0
BEGIN:VEVENT
DTSTAMP:20230105T164630
UID:dev-call.prql-lang.org
DTSTART;TZID=America/New_York:20230108T123000
DTEND;TZID=America/New_York:20230108T130000
RRULE:FREQ=MONTHLY;INTERVAL=1;BYDAY=2SU,4SU;UNTIL=20230331
DESCRIPTION:PRQL Dev Call 2nd & 4th Sunday
LOCATION:Discord #dev-voice-channel\n https://discord.gg/2RUBzz3R3J
SEQUENCE:0
STATUS:CONFIRMED
SUMMARY:PRQL Fortnightly Dev Call
TRANSP:TRANSPARENT
END:VEVENT
END:VCALENDAR
```

**Comment by max-sixty** [comment]
Ha, that works! Very impressive.

If you want to add that to the repo / docs, you're more than welcome to — would be great to have it in a form that folks can click-to-download...

**Comment by max-sixty** [comment]
Also FWIW Google doesn't seem to handle a location field that's only partially a URL; it tries to open a map rather than Discord

**Comment by richb-hanover** [comment]
> Also FWIW Google doesn't seem to handle a location field that's only partially a URL; it tries to open a map rather than Discord

Fixed. I had placed it in the "LOCATION" field, but the proper field is "DESCRIPTION"

**Comment by richb-hanover** [comment]
Let's wait for further comments before adding it to the repo. The .ics file format is extremely fussy - I'm sure this one still has bugs/improvements available

**Comment by richb-hanover** [comment]
> ... on the 1st & 3rd Sunday...

Regrettably, it's the second and fourth Sunday.... :-)

**Comment by max-sixty** [comment]
> > ... on the 1st & 3rd Sunday...
> 
> Regrettably, it's the second and fourth Sunday.... :-)

We've changed it, as we had one this past Sunday, because of the old invite

**Comment by richb-hanover** [comment]
Ahah. Next one is 19 February right? Thanks.

**Comment by snth** [comment]
I just tried to import the current iCal file into Google Calendar and it told me "Imported 0 of 0 events".

Is the file broken?

**Comment by snth** [comment]
@richb-hanover Thank you. I just replaced the file contents with that and it worked now: "Imported 1 of 1 events"

I checked my Google Calendar and it's showing up now.

🙌🙏

**Comment by max-sixty** [comment]
Thanks @richb-hanover ! If you want to PR that, that would be great, otherwise I can. (I saw you mentioned it in the Docker PR; I didn't see the change in the files though)

**Comment by max-sixty** [comment]
FYI this is changing to one hour later, to 10.30AM PT. I updated the description above and #3691 updates the ics
 file.

**Comment by richb-hanover** [comment]
You need to to add `/projects` to both links in the initial post. (Currently, clicking them gives a 404)

Also, I think the PR for the new .ics file is #3961. Thanks.

**Comment by max-sixty** [comment]
> You'll need to to add `/projects` to both links in the initial post. (Currently, clicking them gives a 404)

Thanks a lot! Done

(will delete these posts at some point, hope that's OK)
